_11ty/optimize-html.js

```diff
@@ -79,7 +82,16 @@ const purifyCss = async (rawContent, outputPath) => {
     const after = csso.minify(purged[0].css).css;
     //console.log("CSS reduction", before.length - after.length);
 
-    content = content.replace("</head>", `<style>${after}</style></head>`);
+    // in watch/serve mode, reference the stylesheet. As we bypass 11ty rebuild,
+    // Browsersync will reload only the css file, which is really fast
+    if (isDev) {
```

Comment:
I'm confused by this. Don't you want to do this at the start of this function and then short-circuit the minification?

Reply:
I still wrote the minified css back, but that could be short-circuited too yes .I'll do that